commcare_connect/templates/opportunity/visit_verification.html

```diff
@@ -128,8 +128,13 @@ <h2 class="mb-2">Images</h2>
     </dl>
     {% endif %}
     <div id="action-buttons" style="margin-bottom: 25px;">
-    <a class="btn btn-success" href="{% url 'opportunity:approve_visit' org_slug=request.org.slug pk=visit.id  %}" role="button">Approve</a>
-    <a class="btn btn-danger" href="{% url 'opportunity:reject_visit' org_slug=request.org.slug pk=visit.id  %}" role="button">Reject</a>
+      {% if request.org_membership.is_viewer %}
```

Comment:
Similar to https://github.com/dimagi/commcare-connect/pull/329/files#r1646916211